delta/plugins/composite-views/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/stream/CompositeProgressSuite.scala

```diff
@@ -0,0 +1,52 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.stream
+
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.stream.CompositeBranch.Run
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.nxv
+import ch.epfl.bluebrain.nexus.delta.sourcing.offset.Offset
+import ch.epfl.bluebrain.nexus.delta.sourcing.stream.ProjectionProgress
+import munit.FunSuite
+
+import java.time.Instant
+
+class CompositeProgressSuite extends FunSuite {
+
+  private val source1 = nxv + "source1"
+  private val source2 = nxv + "source1"
+  private val source3 = nxv + "source1"
```

Comment:
I suppose the test will still be successful, but source 1, 2, 3 are defined using "source1"